src/cryptoadvance/specter/server_endpoints/wallets/wallets_api.py

```diff
@@ -523,20 +523,39 @@ def addressinfo(wallet_alias):
             xpubs_descriptor = wallet.get_descriptor(
                 address=address, keep_xpubs=True, to_string=True, with_checksum=True
             )
+            if (not descriptor) or (not xpubs_descriptor):
+                logger.debug(
+                    f"No descriptor or xpubs_descriptor was found for address {address} in wallet {wallet.name}"
+                )
+                return jsonify(success=False)
```

Comment:
I think with https://github.com/cryptoadvance/specter-desktop/pull/2014 we could just raise a SpecterError here since this shouldn't happen and then just check for the error in the client code.

Reply:
However, then this PR would not fix https://github.com/cryptoadvance/specter-desktop/issues/2000  any more:
> In #2000 :
> 
>     * descriptor == ""
> 
>     * xpubs_descriptor == ""
> 
>     * address_info == None
> 
>     * wallet.name is misleading, because the address actually does NOT belong to the wallet

Reply:
Can you write a test perhaps for the concrete issue, still not getting it. Or have you already done this, if so, please point to it. This would also ensure that this PR in the end solves the original issue.

Reply:
See comment below regarding `success`.